cmd/device-util/main.go

```diff
@@ -3,105 +3,41 @@
 package main
 
 import (
-	"errors"
 	"fmt"
+	"log"
 	"os"
 	"strings"
 
-	"github.com/Microsoft/hcsshim/internal/windevice"
-	"github.com/Microsoft/hcsshim/internal/winobjdir"
-	"github.com/urfave/cli"
+	cli "github.com/urfave/cli/v2"
+
+	"github.com/Microsoft/hcsshim/internal/winapi"
```

Comment:
Our usual style is to have one import section for standard library imports, and one for everything else. Suggest we follow that style here.